src/savegame.cpp

```diff
@@ -1489,7 +1489,7 @@ void timed_event_manager::unserialize_all( JsonIn &jsin )
         time_point when;
         int faction_id;
         int strength;
-        tripoint_abs_sm where;
+        tripoint_abs_ms where;
```

Comment:
Btw. this "breaks" existing saves. They will have their submap coordinates loaded as map square coordinates - those they will refer to different locations.